rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,45 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is a powerful utility that enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` was introduced in C++23,
+but many developers are still using C++20 standard. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
```

Comment:
Another part of the story, not yet described in the motivation, is that the implementations of `std::ranges::zip_view` will not work with device policies out of the box. So. our own implementation is necessary not just to provide functionality missing in C++20, but primarily to use it with device policies.

Reply:
I reflected this points (and another points) in the update.